core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala

```diff
@@ -34,25 +34,25 @@ case class StageMetrics(numTasks: Int, duration: String)
 class CollectInformation(apps: Seq[ApplicationInfo]) extends Logging {
 
   def getAppInfo: Seq[AppInfoProfileResults] = {
-    val allRows = apps.map { app =>
+    val allRows = apps.filterNot(_.appInfo == null).map { app =>
```

Comment:
nit: Can we combine `filterNot` and `map` and use `collect()` instead?

Reply:
addressed